react/tic-tac-toe/src/app/screens/Game/utils.js

```diff
@@ -0,0 +1,14 @@
+export function calculateWinner(squares) {
+  const lines = [[0, 1, 2], [3, 4, 5], [6, 7, 8], [0, 3, 6], [1, 4, 7], [2, 5, 8], [0, 4, 8], [2, 4, 6]];
+  for (let i = 0; i < lines.length; i++) {
+    const [a, b, c] = lines[i];
+    if (squares[a] && squares[a] === squares[b] && squares[a] === squares[c]) {
+      return squares[a];
+    }
+  }
+  return null;
+}
+
+export const getWinner = (winner, xIsNext, status) => {
+  return winner ? (status = `Winner:  ${winner}`) : (status = `Next player:  ${xIsNext ? 'X' : 'O'}`);
```

Comment:
Remove braces and return. You can return that ternary directly
Remember to fix the previous comment